code/__HELPERS/roundend.dm

```diff
@@ -188,10 +188,16 @@
 		cb.InvokeAsync()
 	LAZYCLEARLIST(round_end_events)
 
+	var/speed_round = FALSE
+	if(world.time - SSticker.round_start_time <= 300 SECONDS)
+		speed_round = TRUE
+
 	for(var/client/C in GLOB.clients)
 		if(!C.credits)
 			C.RollCredits()
 		C.playtitlemusic(40)
+		if(speed_round)
+			C.give_award(/datum/award/achievement/misc/speed_round, C.mob)
```

Comment:
maybe only give this achievement to someone if they were an antag that round (or alive and within visual range of an antag but not an antag themselves, to discourage suicide wizarding)?

Reply:
whats a quick way to check antag status?

Reply:
Uh, I dunno. Maybe check the code for the roundend report screen, since that lists the round's antags?

Reply:
better yet, just give this achievement to people who haven't suicided?

Reply:
how do you check if a client sudoku'd themselves

Reply:
you can't (as of now) since ghost roles will remove the var.

Reply:
this is fine as is imo, achievements don't have to be things that are hard to do.